### PR TITLE
Fix message list bug

### PR DESCRIPTION
**Pull request by Steven-Chan**
connect #130 

- update skygear sdk version, to fix send image crash app
- fix bug in load more message when the top message is an unsent message

### CODE REVIEW

File: SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift [hunk 1]
```diff
@@ -177,6 +177,19 @@ open class MessageList {
         fatalError("messageIDs contains non String object")
     }
 
+    public func firstSuccessMessage() -> SKYMessage? {
+        var i = 0
+        while (i < self.count) {
```
Comment (rickmak) [inline SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift]: Prefer for loop over while loop.

**Review by rickmak — CHANGES_REQUESTED** [review]
We normally don't use while loop in production code.

**Comment by rickmak** [comment]
lint fails too. Please fix it.

**Comment by Steven-Chan** [comment]
@rickmak updated

**Pull request merged**